cmd/proxy/main.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/go-logr/logr"
 	flag "github.com/spf13/pflag"
 	"gocloud.dev/blob"
+	_ "gocloud.dev/blob/gcsblob"
+	_ "gocloud.dev/blob/s3blob"
```

Comment:
The drivers needed to be imported in the proxy rather than the restore-agent

```
kubectl -n eco-system logs deploy/eco-proxy --follow
2020-02-26T14:57:02.588Z	INFO	setup	Starting proxy	{"version": "v0.2.0-15-gb1deda1"}
2020-02-26T14:57:02.588Z	INFO	setup	Listening	{"grpc-address": ":8080"}
2020-02-26T14:57:29.585Z	ERROR	proxy-server.download	failed to open bucket	{"req-backup-url": "s3://backups.test.improbable.io/snapshot.db?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true", "error": "open .Bucket: no driver registered for \"s3\" for URL \"s3://backups.test.improbable.io?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true\"; available schemes: "}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.loggedError
	/workspace/cmd/proxy/main.go:46
main.(*proxyServer).Download
	/workspace/cmd/proxy/main.go:59
github.com/improbable-eng/etcd-cluster-operator/api/proxy/v1._ProxyService_Download_Handler
	/workspace/api/proxy/v1/proxy.pb.go:334
google.golang.org/grpc.(*Server).processUnaryRPC
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:995
google.golang.org/grpc.(*Server).handleStream
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:1275
google.golang.org/grpc.(*Server).serveStreams.func1.1
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:710
2020-02-26T14:57:31.883Z	ERROR	proxy-server.download	failed to open bucket	{"req-backup-url": "s3://backups.test.improbable.io/snapshot.db?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true", "error": "open .Bucket: no driver registered for \"s3\" for URL \"s3://backups.test.improbable.io?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true\"; available schemes: "}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.loggedError
	/workspace/cmd/proxy/main.go:46
main.(*proxyServer).Download
	/workspace/cmd/proxy/main.go:59
github.com/improbable-eng/etcd-cluster-operator/api/proxy/v1._ProxyService_Download_Handler
	/workspace/api/proxy/v1/proxy.pb.go:334
google.golang.org/grpc.(*Server).processUnaryRPC
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:995
google.golang.org/grpc.(*Server).handleStream
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:1275
google.golang.org/grpc.(*Server).serveStreams.func1.1
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:710
2020-02-26T14:57:33.986Z	ERROR	proxy-server.download	failed to open bucket	{"req-backup-url": "s3://backups.test.improbable.io/snapshot.db?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true", "error": "open .Bucket: no driver registered for \"s3\" for URL \"s3://backups.test.improbable.io?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true\"; available schemes: "}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128
main.loggedError
	/workspace/cmd/proxy/main.go:46
main.(*proxyServer).Download
	/workspace/cmd/proxy/main.go:59
github.com/improbable-eng/etcd-cluster-operator/api/proxy/v1._ProxyService_Download_Handler
	/workspace/api/proxy/v1/proxy.pb.go:334
google.golang.org/grpc.(*Server).processUnaryRPC
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:995
google.golang.org/grpc.(*Server).handleStream
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:1275
google.golang.org/grpc.(*Server).serveStreams.func1.1
	/go/pkg/mod/google.golang.org/grpc@v1.23.1/server.go:710

```